### ISSUE TITLE
Issue in handling of service & service linked roles during environment creation

### ISSUE THREAD

**Issue by blitzmohit** [awslabs/aws-dataall#695]
### Describe the bug

As part of the env creation a user can provide IAM Role Name on the UI. 
If the role name for a service or service linked role is provided during the env creation, the stack fails.

### How to Reproduce

```
Taking an example service role:
   Role name: QuickSightAction
   Role ARN: arn:aws:iam::123456789012:role/service-role/QuickSightAction
```
Create a new environment with any role that is of the format with arn that includes a "/"

### Expected behavior

Env stack should succeed

### OS

Linux

### Python version

python3.8

### AWS data.all version

v2.0.0-beta1

### Additional context

Based on discussion with @noah-paige we could instead modify the input such that the role arn be provided instead of role name since role name does not include the complete information.

**Comment by dlpzx** [comment]
Yes, thanks @blitzmohit. We are thinking of including the fix as part of V2.0.0 or in the next patch release

**Comment by dlpzx** [comment]
Merged and released with v2.1.0 🚀
